packages/design-system/src/types/form.ts

```diff
@@ -15,7 +15,7 @@ export type IFormInput = {
 	initialValue?: string | number | boolean | null;
 	properties: {
 		label?: string;
-		type?: 'text' | 'email' | 'password' | 'select' | 'multi-select' | 'info';
```

Comment:
is this needed still?

Reply:
Not sure. I think there was en error in the console saying that `checkbox` type is not valid until I added it here.